crates/sui-transactional-test-runner/src/programmable_transaction_test_parser/parser.rs

```diff
@@ -38,7 +36,7 @@ pub struct ParsedMoveCall {
     pub package: Identifier,
     pub module: Identifier,
     pub function: Identifier,
-    pub type_arguments: Vec<ParsedType>,
+    pub type_arguments: Vec<String>,
```

Comment:
This feels a bit worse to me. Why the change?

Reply:
This is because we are deferring the parsing of the types till the callsite when we have the named addresses in scope. I think I can parse it directly though. Let me take a look!